### PR TITLE
Add default journal support message to press manager view and template

### LINKED ISSUE #2673: Add support contact message to press manager
Do this but for the press mananger: https://github.com/BirkbeckCTP/janeway/issues/2586

### PR DESCRIPTION
**Pull request by joemull**
Fixes #2673

### CODE REVIEW

File: src/core/logic.py [hunk 1]
```diff
@@ -846,7 +846,7 @@ def render_nested_setting(
     setting = setting_handler.get_setting(
         setting_group,
         setting_name,
-        request.journal
+        request.journal if request.journal else None,
```
Comment (mauromsl) [inline src/core/logic.py:849]: Is this change necessary?

If request.journal is None, then there is no need for the if statement.
This would only be relevant if request.journal was set to `False` or a Falsey value like empty string or empty list

**Review by mauromsl — COMMENTED** [review]
Just a question

File: src/core/logic.py (hunk 1 above)
Comment (joemull) [reply src/core/logic.py:849]: I wasn't sure about how the request object was created and whether it would have been assigned anything for `request.journal` at the press level, and figuring that out would have taken some time. If you say it's not needed, then I'll take out the line.

For future work with request objects, where in the codebase are they created?

Comment (ajrbyers) [reply src/core/logic.py:849]: request.journal and request.press always exist in the request/response cycle and are set in `core.middleware`.

**Review by mauromsl — APPROVED** [review]
